File: src/main/java/org/kite9/diagram/batik/transform/NoopTransformer.java
package org.kite9.diagram.batik.transform;

import org.kite9.diagram.dom.painter.Painter;
import org.kite9.diagram.dom.processors.XMLProcessor;
import org.w3c.dom.Document;
import org.w3c.dom.Element;

public class NoopTransformer implements SVGTransformer {

	@Override
	public Element postProcess(Painter p, Document d, XMLProcessor posProcessor) {	
		return p.output(d, posProcessor);
	}

}

File: src/main/kotlin/org/kite9/diagram/dom/transform/NoopTransformer.kt
package org.kite9.diagram.dom.transform

import org.kite9.diagram.dom.painter.Painter
import org.kite9.diagram.dom.processors.XMLProcessor
import org.w3c.dom.Document
import org.w3c.dom.Element

class NoopTransformer : SVGTransformer {

    override fun postProcess(p: Painter, d: Document, postProcessor: XMLProcessor): Element? {
        return p.output(d, postProcessor)
    }
}